### PR TITLE
Import transform API

### PR DESCRIPTION
**Pull request by ahoffer**
Use the transform service's API classes

### CODE REVIEW

**Comment by ahoffer** [comment]
build now.

File: ingest/src/main/java/com/connexta/ingest/transform/TransformClient.java [hunk 1]
```diff
@@ -6,53 +6,44 @@
...
 
-  public TransformResponse requestTransform(TransformRequest transformRequest) {
-    ResponseErrorHandler originalRequestHandler = restTemplate.getErrorHandler();
-    try {
-      restTemplate.setErrorHandler(new NoOpResponseErrorHandler());
-      return postForTransform(transformRequest);
-    } finally {
-      restTemplate.setErrorHandler(originalRequestHandler);
-    }
+  private URI transformEndpoint;
```
Comment (vinamartin) [inline ingest/src/main/java/com/connexta/ingest/transform/TransformClient.java]: ❓ I'm a little confused on how this value gets injected. 
I don't see any autowiring or anything setting this through dependency injection

Comment (ahoffer) [reply ingest/src/main/java/com/connexta/ingest/transform/TransformClient.java]: We are not as far as injecting it. We are manually setting it for now.

**Comment by emmberk** [comment]
build now

File: ingest/src/main/java/com/connexta/ingest/transform/TransformClient.java [hunk 2]
```diff
@@ -6,53 +6,52 @@
...
-  public TransformClient(RestTemplate restTemplate) throws URISyntaxException {
-    this.restTemplate = restTemplate;
+  @Autowired private RestTemplate restTemplate;
+
+  private URI transformEndpoint;
+
+  @PostConstruct
+  private void setEndpoint() throws URISyntaxException {
+    // TODO: Set endpoint URL
+    setTransformEndpoint("http://TODO");
```
Comment (Kjames5269) [inline ingest/src/main/java/com/connexta/ingest/transform/TransformClient.java]: ✏️ This endpoint can come from the configuration file

File: ingest/src/main/java/com/connexta/ingest/service/impl/IngestServiceImpl.java [hunk 3]
```diff
@@ -33,6 +37,17 @@ public UUID ingest(
...
+    transformRequest.setCallbackUrl("http://blah/blah");
+    transformRequest.setId("1");
+    transformRequest.setMimeType("nitf");
+    transformRequest.setProductLocation("prod");
+    transformRequest.setStagedLocation("stage");
+
+    TransformResponse transformResponse = transformClient.requestTransform(transformRequest);
+
+    LOGGER.warn("Completed transform request, reponse is {}", transformResponse);
     return s3Adaptor.upload(ingestRequest);
```
Comment (emmberk) [inline ingest/src/main/java/com/connexta/ingest/service/impl/IngestServiceImpl.java]: ❗️Upload to S3 before sending the transformation request

File: ingest/src/main/java/com/connexta/ingest/transform/TransformClient.java (hunk 2 above)
Comment (ahoffer) [reply ingest/src/main/java/com/connexta/ingest/transform/TransformClient.java]: >  This endpoint can come from the configuration file

Do you mean the application.yml file?

Comment (vinamartin) [reply ingest/src/main/java/com/connexta/ingest/transform/TransformClient.java]: Yeah it seems like you should inject this.

Comment (emmberk) [reply ingest/src/main/java/com/connexta/ingest/transform/TransformClient.java]: We can address this in a future commit. See https://github.com/connexta/multi-int-store/pull/47.

**Comment by emmberk** [comment]
build now

**Comment by emmberk** [comment]
`./gradlew dependencyCheckAnalyze --info` succeeded for me locally. I also manually heroed this PR.

**Comment by emmberk** [comment]
I rebased. `./gradlew clean build` succeeded for me locally.

build now

**Comment by emmberk** [comment]
The build failed because of a known issue with downloading CVEs. `./gradlew dependencyCheckAnalyze --info` succeeded for me locally.

**Pull request merged**